### ISSUE TITLE
Possible error in Cannon run script

### ISSUE THREAD

**Issue by sdeastham** [geoschem/geos-chem#1685]
In the Harvard Cannon GCHP run script, there's a line which I think is meant to try and modify the core distribution: https://github.com/geoschem/geos-chem/blob/4722f288e90291ba904222f4bbe4fc216d17c34a/run/GCHP/runScriptSamples/operational_examples/harvard_cannon/gchp.run#L34

However, this line actually ends up being a comparison, as the left side is formatted to return (not set) the variable. This can be easily fixed as 

`planeCount=$(( ${planeCount} + 1 ))`

This may well have been a bug I introduced in the first place, so hopefully it hasn't caused others too many issues!

**Comment by sdeastham** [comment]
Great - thanks @yantosca! Tagging @mcdonh who was the first one to identify this issue.

**Comment by lizziel** [comment]
I think this is actually my bad. I'll tag this fix as 14.2.0 to go into the next version. @yantosca, I will assign you. The bug is also in the script for NASA Pleiades.

Thanks @sdeastham for finding this! I wonder what the performance impact is. Have you tested that?

**Comment by yantosca** [comment]
@lizziel: this is also in the GCHP integration test script, but I don't think it matters since we run those at c24.  I'll fix that at the same time.

**Comment by yantosca** [comment]
So I wrote a test script to try to see when the `planeCt + 1` would be used.  It doesn't happen that much:

```bash
#!/bin/bash

NX=4
NY=24
SLURM_NNODES=7

let coreCt=$(( ${NX} * ${NY} ))
echo "coreCt: $coreCt"

let planeCt=$(( ${coreCt} / ${SLURM_NNODES} ))
echo "planeCt 1: $planeCt"

[[ $(( ${coreCt} % ${SLURM_NNODES} )) > 0 ]] && let planeCt+=1
echo "planeCt 2: $planeCt"
```
This gave the following output:
```console
coreCt: 96
planeCt 1: 13
planeCt 2: 14
```
But for most situations I don't think that planeCt would be updated. 

For `NX=6`, `NY=1`, `SLURM_NNODES=1`  we get `coreCt = planeCt = 6`.

For `NX=4`, `NY=24`, SLURM_NNODES=2` (the old benchmark setup) we get `coreCt=96` and `planeCt=48`, etc.
 
So even though this was a bug in the script, I don't think the bug executed very much.

I'll make a PR with the fix.

**Comment by msulprizio** [comment]
The pull request (#1686) for this issue has now been merged into the 14.2.0 development branch.
